tasks.go

```diff
@@ -46,6 +53,11 @@ func RunAllTasks(runner Runner, path string, workers int) {
 		fmt.Fprint(os.Stderr, ".")
 	}
 	fmt.Fprintln(os.Stderr)
+
+	delta := time.Since(start)
+	// Remove sub-second precision.
+	delta -= delta % time.Second
+	log.Printf("Run %d tasks in %v.", taskCount, delta)
```

Comment:
We could also add a number of projects here, to help with making sure no tasks are missing.